cmd/test-utils_test.go

```diff
@@ -1975,6 +1975,7 @@ func ExecObjectLayerTest(t TestErrHandler, objTest objTestType) {
 	if err != nil {
 		t.Fatalf("Initialization of object layer failed for single node setup: %s", err)
 	}
+
```

Comment:
My recommendation is to avoid unrelated diffs

Reply:
Removing..